packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -37,6 +40,14 @@ interface IButtonClassNames {
 
 export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState> implements IButton {
 
+  private get _isSplitButton(): boolean {
```

Comment:
I think that's currently already happening. From the menuProps' documentation:
`Props for button menu. Providing this will default to showing the menu icon. See menuIconProps for overriding how the default icon looks. Providing this in addition of onClick will render a SplitButton.`  
Are you talking about something else?